lib/components/atoms/Video/Video.style.js

```diff
@@ -0,0 +1,6 @@
+import { css } from 'styled-components';
+
+export default css`
+  width: 50%;
```

Comment:
Let it be 100%

Reply:
Its the variation of video with style and updated. This value will be come from props